File: src/main/java/org/zenbot/timetableupdater/domain/Schedule.java
package org.zenbot.timetableupdater.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Data
@Document
public class Schedule {

    @Id
    private String id;
    private List<BusArrival> busArrivals;
}

File: src/main/kotlin/org/zenbot/timetableupdater/domain/Schedule.kt
package org.zenbot.timetableupdater.domain

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
data class Schedule (
        @Id
        var id: String? = null,
        var busArrivals: List<BusArrival> = emptyList()
)